plugin/src/main/java/io/jenkins/plugins/casc/core/HudsonPrivateSecurityRealmConfigurator.java

```diff
@@ -8,7 +8,11 @@
 import org.kohsuke.accmod.Restricted;
 import org.kohsuke.accmod.restrictions.NoExternalUse;
 import org.kohsuke.stapler.DataBoundConstructor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Please don't use `org.slf4j.Logger`  
So far we have been using `java.util.logging.Logger`

If you want us to switch to slf4j that requires a separate PR :sweat_smile:

Reply:
Wow...I've never _not_ used slf4j.